tools/pepquery/pepquery.xml

```diff
@@ -0,0 +1,196 @@
+<tool id="pepquery" name="PepQuery" version="@VERSION@">
+    <description>Peptide-centric search engine for novel peptide identification and validation.</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <requirements> 
+        <requirement type="package" version="@VERSION@">pepquery</requirement>
+    </requirements>
+    <command detect_errors="exit_code">
+        <![CDATA[
+            cp $req_inputs.spectrum_file ${req_inputs.spectrum_file}.mgf &&
```

Comment:
This means you copy the file to the Galaxy database, thats not a good idea. Pleaes copy, or better link them to the current working dir, e.g. `./`